devnets/7420/erigon/erigon.toml

```diff
@@ -17,6 +17,7 @@ http = true
 "authrpc.vhosts" = "*"
 "snapshots" = false
 "prune" = "htc"
+"private.api.addr" = "localhost:9091"
```

Comment:
9091 is alertmanager port (another basic svc that comes with prometheus).

Reply:
also, 127.0.0.1 is probably better than localhost (to keep the same approach as the default value)

Reply:
Switched to 9098